pyaerocom/aeroval/coldatatojson_engine.py

```diff
@@ -869,7 +970,8 @@ def _select_period_season_coldata(coldata, period, season):
     return ColocatedData(arr)
 
 def _process_heatmap_data(data, region_ids, use_weights, use_country,
-                          meta_glob, periods, seasons):
+                          meta_glob, periods, seasons, add_trends=False,
+                          trends_min_yrs=7):
```

Comment:
Please remove default values here.
```suggestion
                          meta_glob, periods, seasons, add_trends,
                          trends_min_yrs):
```